packages/target-nodejs-sdk/src/index.js

```diff
@@ -231,12 +241,23 @@ export default function bootstrap(fetchApi) {
         },
         logger: this.logger,
         useBeacon: true,
-        ...options
+        ...request
       };
 
-      return executeDelivery(targetOptions).then(
-        preserveLocationHint.bind(this)
-      );
+      return executeDelivery(
+        targetOptions,
+        undefined,
+        this.telemetryProvider
+      ).then(preserveLocationHint.bind(this));
+    }
+
+    executeTelemetries(deliveryRequest, telemetryEntries) {
```

Comment:
Let's move this fn to target-nodejs-sdk/helper.js to keep logic inside index.js as minimal as possible

Reply:
that sounds a lot better